### DISCUSSION TITLE
If u help me, I will give u reward.

### DISCUSSION THREAD

**Discussion by flybear5500** [SYSTRAN/faster-whisper#1111] [Q&A]
Recently I did STT on Arabic voice file. The result language is Arabic but the text is English. 
I want Arabic text. Can you tell me how to set up the model?

**Comment by techspecshq** [comment]
> Recently I did STT on Arabic voice file. The result language is Arabic but the text is English. I want Arabic text. Can you tell me how to set up the model?

Can you upload the audio so that I can try it on my end?
